targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TARGET_MODULE_UBLOX_ODIN_W2/sdk/ublox-odin-w2-drivers/OdinWiFiInterface.h

```diff
@@ -70,7 +78,16 @@ class OdinWiFiInterface : public WiFiInterface, public EMACInterface
      *  @return          0 on success, or error code on failure
      */
     virtual nsapi_error_t set_credentials(const char *ssid, const char *pass, nsapi_security_t security = NSAPI_SECURITY_NONE);
-    
+
+    /** Set the WiFi network credentials
+     *
+     *  @param client_cert      Pointer to client certificate
+     *  @param clitne_pvt_key   Pointer to client private key
```

Comment:
[minor] Just a typo, "clitne" instead of "client", here and in the actual variable name as well.